### ISSUE TITLE
Investigate `save;${string}: ${string}` string in container-runtime

### ISSUE THREAD

**Issue by sdeshpande3** [microsoft/FluidFramework#8288]

**Comment by scottn12** [comment]
The save operation type was deprecated since its introduction in PR https://github.com/microsoft/FluidFramework/pull/6432. The deprecation comment discourages the use of save operations and does not suggest any replacement. 

This save operation type is only found in the `handleOp` function (`packages/runtime/container-runtime/src/runningSummarizer.ts`):

```ts
public handleOp(error: any, { sequenceNumber, type, clientId, contents }: ISequencedDocumentMessage) {
    if (error !== undefined) {
        return;
    }
    this.heuristicData.lastOpSequenceNumber = sequenceNumber;

    if (this.tryRunEnqueuedSummary()) {
        // Intentionally do nothing; check for enqueued on-demand summaries
    } else if (type === MessageType.Save) {
        // Check for ops requesting summary
        // Note: as const is only required until TypeScript version 4.3
        this.trySummarize(`save;${clientId}: ${contents}` as const);
    } else {
        this.heuristicRunner?.run();
    }
}
```

It appears the individual who introduced this operation type has left the project. @heliocliu or @ChumpChief , would either of you have any insight/additional context as to why we should avoid save operation types, and how we should handle them moving forward?

**Comment by ChumpChief** [comment]
I don't have context, maybe @vladsud does?  I'm not aware of a time when we triggered summaries via ops, looks like remaining usage is some dead test code.

It looks like `MessageType.Save` should also be deprecated/removed along with it if this is removed.

**Comment by scottn12** [comment]
I definitely agree that we should deprecate/remove `MessageType.Save` as well if we go forward removing the save operation completely. The only other use of this enum is here: https://github.com/microsoft/FluidFramework/blob/71304171edf46d8259b8eea055cf0c7cce123a6d/server/routerlicious/packages/test-utils/src/messageFactory.ts#L157
The above line is in a function called `createSave()` which appears to not be used anywhere in FF repo or our partner repos.

**Comment by vladsud** [comment]
I do not think we ever used MessageType.Save flow. I mean - code was probably there, but I do not think any scenario was ever dependent on it.
There are other ops there that are questionable / should probably be removed. 
RemoteHelp, RoundTrip seems to be not used at all.
I know nothing about Control.
